### PR TITLE
Adapted changes

### PR DESCRIPTION
**Pull request by Kreyenschmidt**
Hello, 

would you like to include my ontology to your repository, thank you. 

Best regards, Christian

### CODE REVIEW

File: wdio/.htaccess [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Turn off MultiViews
+Options -MultiViews
+
+# Directive to ensure *.rdf files served as appropriate content type,
+# if not present in main apache config
+AddType application/rdf+xml .rdf
+AddType application/rdf+xml .owl
+AddType text/turtle .ttl
+AddType application/n-triples .n3
+AddType application/ld+json .json
```
Comment (dgarijo) [inline wdio/.htaccess:10]: Why adding the types if you don't do content negotiation with them?

**Comment by dgarijo** [comment]
@Kreyenschmidt I left you a small comment. Can you please add your github id in the readme file? Thanks

File: wdio/readme.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+https://kreyenschmidt.github.io/wdio/
+
+Author:
+
+**Christian Kreyenschmidt**  
+_Research associate_    
+_Institute for Database-oriented Engineering_   
+_Jade University of Applied Sciences in  Oldenburg_     
+<christian.kreyenschmidt@jade-hs.de>
+_Github ID: 87878109_
```
Comment (TallTed) [inline wdio/readme.md:13]:
```suggestion
<christian.kreyenschmidt@jade-hs.de>  
_Github ID: @Kreyenschmidt_
```

[the remark this answers is not in the document]
Comment (Kreyenschmidt) [reply wdio/readme.md]: Ok, thank you.

**Pull request merged**